src/parse.js

```diff
@@ -70,6 +69,23 @@ export const areBlocksEqual = curry((a, b) =>
   (a.msgid === b.msgid && a.msgctxt === b.msgctxt)
 );
 
+/**
+ * Returns whether two gettext reference comment are considered equal
+ */
+const areReferenceEqual = curry((a, b) =>
```

Comment:
This ought to be either `areReferencesEqual` or `isReferenceEqual`. Doesn't really matter which!

Reply:
Fixed the name :)